scipy/optimize/_zeros_py.py

```diff
@@ -1396,6 +1398,298 @@ def toms748(f, a, b, args=(), k=1,
     return _results_select(full_output, (x, function_calls, iterations, flag))
 
 
+def _bracket_root_iv(func, a, b, min, max, factor, args, maxiter):
+
+    if not callable(func):
+        raise ValueError('`func` must be callable.')
+
+    if not np.iterable(args):
+        args = (args,)
+
+    a = np.asarray(a)[()]
+    if not np.issubdtype(a.dtype, np.number) or np.iscomplex(a).any():
+        raise ValueError('`a` must be numeric and real.')
+
+    b = a + 1 if b is None else b
+    min = -np.inf if min is None else min
+    max = np.inf if max is None else max
+    # golden ratio as suggested in gh-18348, but higher values are faster.
```

Comment:
Seeing Warren's point, I would just use 2.

Reply:
I think it can be proven, under certain assumptions, that larger numbers tend to be better, because of the way these brackets expand and how rootfinders converge. So I'm happy to do that, but I was just going with the flow here. I think it was 2 or 10 originally.

Reply:
Changed back to `2`